### PR TITLE
Fix Bug 1213911: Enable Czech 1.1 & 1.3

### PR DESCRIPTION
**Pull request by bensternthal**
Enabling 1.1 & 1.3, note 1.3 was never pending but the translations are in.

### CODE REVIEW

**Comment by glogiotatidis** [comment]
Test this please

File: masterfirefoxos/settings/base.py [hunk 1]
```diff
@@ -224,12 +224,12 @@
...
     'slug': '1-1',
-    'locales': ['el', 'en', 'es', 'it', 'pt-br'],
+    'locales': ['cs', 'el', 'en', 'es', 'it', 'pt-br'],
     'pending_locales': [
-        'hr', 'cs', 'de', 'hu', 'pl', 'sr', 'xx']}
+        'hr', 'de', 'hu', 'pl', 'sr', 'xx']}
 VERSIONS_LOCALE_MAP['1.3'] = {
     'slug': '1-3',
-    'locales': ['en', 'es'],
+    'locales': ['cs', 'en', 'es'],
```
Comment (glogiotatidis) [inline masterfirefoxos/settings/base.py]: Hmmm `cs` was not in the `pending_locales` for 1.3 version. Probably we should enable that first and make sure that all strings are translated.

**Comment by bensternthal** [comment]
Good point, will resubmit.

**Comment by bensternthal** [comment]
Test this please

**Comment by bensternthal** [comment]
@glogiotatidis are we blocked from testing on the l10n site?

**Comment by glogiotatidis** [comment]
Yes, I'll complete the move to the new cluster this week and I'll unblock this PR.

**Comment by glogiotatidis** [comment]
Test this please

**Comment by glogiotatidis** [comment]
Thanks @bensternthal :cake:

**Pull request merged**